src/runtime_src/xdp/profile/database/static_info_database.cpp

```diff
@@ -2458,6 +2530,15 @@ namespace xdp {
     } else {
       // This is a previously used device being reloaded with a new xclbin
       devInfo = itr->second.get();
+      ConfigInfo *config = devInfo->currentConfig();
+      if (config && (config->type == CONFIG_PL_DEVICE_INTF_ONLY) &&
+          (xclbinType == XCLBIN_PL_ONLY || xclbinType == XCLBIN_AIE_PL)) {
+          std::stringstream errMsg;
+          errMsg << "Debug and Profiling features are not supported if PL xclbin hw_context is created after AIE only xclbin hw_context for device. ";
```

Comment:
CONFIG_PL_DEVICE_INTF_ONLY is created for AIE only xclbin only when AIE Trace is enabled.
```suggestion
          errMsg << "AIE Trace is not supported if PL xclbin hw_context is created after AIE only xclbin hw_context for device. ";
```